src/test/ui/parser/keyword-const-as-identifier.rs

```diff
@@ -1,5 +1,6 @@
 // This file was auto-generated using 'src/etc/generate-keyword-tests.py const'
 
-fn main() {
-    let const = "foo"; //~ error: expected identifier, found keyword `const`
```

Comment:
can you check whether your latest changes make the orginal test work again?

Reply:
```
error: expected `{`, found `=`
 --> src/test/ui/parser/keyword-const-as-identifier.rs:4:15
  |
4 |     let const = "foo"; //~ error: expected identifier, found keyword `const`
  |               ^ expected `{`

```

Reply:
Not sure. If we could add an additional note to this specific error, informing the user about const patterns, then that would suffice I believe